doc/index.rst

```diff
@@ -29,23 +34,21 @@ PennyLane-Lightning provides the following device:
 
    installation
    support
-   Architecture <arch/arch_overview>
 
 .. toctree::
    :maxdepth: 2
    :caption: Usage
    :hidden:
 
-   devices
+   lightning_qubit/device
+   lightning_kokkos/device
 
 .. toctree::
    :maxdepth: 2
    :caption: Development
    :hidden:
 
-   add_kernel
-   benchmark
-   avx_kernels/index
+   lightning_qubit/development/index
 
```

Comment:
@AmintorDusko I can't comment on it below, but the `code/__init__` file (which leads to this page https://xanaduai-pennylane--473.com.readthedocs.build/projects/lightning/en/473/code/__init__.html) is only showing `LightningQubit`, and not `LightningKokkos`?